packages/tupaia-web/src/constants/constants.ts

```diff
@@ -3,9 +3,6 @@
  * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
  */
 
-export const DEFAULT_PROJECT_CODE = 'explore';
-export const DEFAULT_ENTITY_CODE = 'explore';
-
```

Comment:
these consts are only used in the default url so I'm just inlining them below as I think the concept of default project and entity doesn't make sense outside of the default url